homeassistant/components/litterrobot/quality_scale.yaml

```diff
@@ -0,0 +1,86 @@
+rules:
+  # Bronze
+  action-setup:
+    status: todo
+    comment: |
+      Action async_set_sleep_mode is currently setup in the vacuum platform
+  appropriate-polling:
+    status: done
+    comment: |
+      Primarily relies on push data, but polls every 5 minutes for missed updates
+  brands: done
+  common-modules:
+    status: todo
+    comment: |
+      hub.py should be renamed to coordinator.py and updated accordingly
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions:
+    status: todo
+    comment: Can be finished after async_set_sleep_mode is moved to async_setup
+  docs-high-level-description: done
+  docs-installation-instructions: todo
+  docs-removal-instructions: todo
+  entity-event-setup: done
```

Comment:
I think this is interesting because you subscribe to both the coordinator and the robot itself. I am wondering what the use case for both is